test/mpi/errors/comm/Makefile.am

```diff
@@ -12,5 +12,6 @@ EXTRA_DIST = testlist
 ## for all programs that are just built from the single corresponding source
 ## file, we don't need per-target _SOURCES rules, automake will infer them
 ## correctly
-noinst_PROGRAMS = cfree ccreate1 manysplit userdup too_many_comms too_many_comms2 too_many_comms3 too_many_icomms too_many_icomms2
-
+noinst_PROGRAMS = cfree ccreate1 manysplit userdup too_many_comms too_many_comms2 too_many_comms3 too_many_icomms too_many_icomms2 \
```

Comment:
@Bayyapu I thought we had agreed that new tests were going to be named `<function>_nullarg.c`. It looks like this version of the PR names them all `<function>_test.c`.